source/stylesheets/base/_typography.scss

```diff
@@ -12,15 +12,15 @@ body {
 a {
   text-decoration: none;
   color: $orange-darker;
-  &:hover, :focus {
+  &:hover, &:focus {
     color: $orange-darkest;
   }
 }
 p {
   a {
     color: $orange-darker;
     text-decoration: underline;
-    &:hover, :focus {
+    &:hover, &:focus {
```

Comment:
This was pretty interesting, the focus selector here was unreachable since the css attribute looked like `a :focus` instead of `a:focus` this meant the bootstrap `a:focus` was being used to style the link blue